integration/bug-report-test.ts

```diff
@@ -47,30 +47,19 @@ test.beforeAll(async () => {
     // `createFixture` will make an app and run your tests against it.
     ////////////////////////////////////////////////////////////////////////////
     files: {
-      "app/routes/index.jsx": js`
```

Comment:
This test file is used as a template. Please move this test to `integration/form-test.ts`

Reply:
Oopsie! 😊